Userland/Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -973,6 +973,7 @@ static ErrorOr<void> read_restart_interval(JPEGStream& stream, JPEGLoadingContex
         return Error::from_string_literal("Malformed DRI marker found");
     }
     context.dc_restart_interval = TRY(stream.read_u16());
+    dbgln_if(JPEG_DEBUG, "Restart marker: {}", context.dc_restart_interval);
```

Comment:
Out if interest, what is this for? We don't use these for anything, and it's kinda noisy to dump.

Reply:
`context.dc_restart_interval` is used in `decode_huffman_stream`
I'm not sure to understand you